src/storage/hummock_test/benches/bench_hummock_iter.rs

```diff
@@ -70,10 +70,14 @@ fn criterion_benchmark(c: &mut Criterion) {
         .unwrap()
     });
 
-    let hummock_storage =
-        runtime.block_on(async { global_hummock_storage.new_local(TableId::default()).await });
+    let mut hummock_storage = runtime.block_on(async {
```

Comment:
May off-topic: we can enable `async` support for criterion instead of calling `block_on` for each future.
https://bheisler.github.io/criterion.rs/book/user_guide/benchmarking_async.html